daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -1868,6 +1868,61 @@ char     *QDECL PRINTF_LIKE(1) va( const char *format, ... )
 =====================================================================
 */
 
+std::string InfoMapToString( const InfoMap& map )
+{
+	std::string info_string;
+
+	for ( const auto& pair : map )
+	{
```

Comment:
Unnecessary. There is code to iterate infostrings already. Info_NextPair. See cg_rocket_datasource.cpp:CG_Rocket_BuildServerInfo() for example usage.